src/index.tsx

```diff
@@ -119,6 +123,7 @@ interface BaseControlDefinition {
   errorMessages?: {
     [key: string]: ErrorMessage;
   };
+  multiple?: boolean;
```

Comment:
Added a `multiple` property to all inputs, so the user can specify that they're going to render multiple inputs with the same name.

We need this property as we can't look at the DOM from our TS typings.